feat(docs) Aligning with commit lag change for 6.6 (#1197)
questdb/questdb.io: docs/introduction.md

- [Binaries](/docs/get-started/binaries/) for direct downloads to run on Linux,
  macOS or Windows
- [Homebrew](/docs/get-started/homebrew/) for running QuestDB on macOS
- [Create your first database](/docs/get-started/first-database/) provides a

>   guide for once QuestDB is running

Reviewer: I think we need to delete "for" ?

Author: ```suggestion
  guide once QuestDB is running
```

utilities for collecting metrics and visualizing data:

- [Prometheus](/docs/third-party-tools/prometheus) monitoring and alerting.
- [Grafana](/docs/third-party-tools/grafana) instructions for connecting QuestDB
  as a datasource for building visualizations and dashboards
- [Kafka](/docs/third-party-tools/kafka/overview) guide for ingesting data from

>   topics into QuestDB

Reviewer: from topics or from Kafka topics?

- [SQL extensions](/docs/concept/sql-extensions) allow performant time series
  analysis with a concise syntax

> - [JIT compiler](/docs/concept/jit-compiler) to speed up queries

Reviewer: should it not be "WHERE" queries specifically?

Author: Sorry I don't understand what you meant, this is reflecting the sidebar so the items are fixed?

Reviewer: - [JIT compiler](/docs/concept/jit-compiler) to speed up queries

my question is: should we be more specific, saying that we speed "WHERE" queries ? its a question btw, not asking you to do it

Author: ```suggestion
- [JIT compiler](/docs/concept/jit-compiler) to speed up `WHERE` clauses
```

- [Partitions](/docs/concept/partitions) can be used for significant performance

>   benefits on calculations and queries

Reviewer: this is a bit convoluted. what about: 
should be used to lift less data for time-based queries

Author: I find "list less data" very vague but agree it could be simplified -edited your suggestion. I will keep "can" too because it is a suggestion.

> - [Partitions](/docs/concept/partitions) can be used for significant performance
>   benefits on calculations and queries

Author: ```suggestion
- [Partitions](/docs/concept/partitions) can be used to perform time-based queries more efficiently
```
